lib/alerts_viewer_web/live/alerts_to_close_live.html.heex

```diff
@@ -0,0 +1,52 @@
+<.header>
+  Canidates to Close
+</.header>
+
+<div class="flex gap-8">
+  <.controls_form phx-change="select-algorithm">
+    <.input
+      type="select"
+      name="algorithm"
+      label="Algorithm"
+      value={@current_algorithm}
+      options={@algorithm_options}
+    />
+  </.controls_form>
+
+  <.live_component
+    module={@current_algorithm}
+    id={:algorithm_controls}
+    alerts_by_route={@alerts_by_route}
+    stats_by_route={@stats_by_route}
+  />
+</div>
+
+<.table id="bus_alerts" sticky={true} rows={@alerts_by_route |> Keyword.keys()}>
```

Comment:
```suggestion
<.table id="bus_alerts" sticky={true} rows={Keyword.keys(@alerts_by_route)}>
```
suggestion: I think it's considered best practice to not use a pipe chain for a single call.